### DISCUSSION TITLE
Security policy for responsible disclosure

### DISCUSSION THREAD

**Discussion by camallen** [simonw/datasette#1777] [General]
There is no security policy https://github.com/simonw/datasette/security/policy configured on this repo and I have not found a place to responsibly report security issues either via this repository or via the documentation site (https://datasette.io/-/beta?q=security)

How should folks responsibly disclose / report any security issues they find with datasette?

**Comment by brandonrobertz** [comment]
Good question. You should probably try and raise a GH issue or get ahold of Simon in some other way.
